Adding what's new entry to forward alerts

dockerymick opened the pull request:
https://github.com/open-cluster-management/backlog/issues/11194
https://github.com/open-cluster-management/backlog/issues/12110
https://github.com/open-cluster-management/backlog/issues/13813

dockerymick committed:
Adding what's new entry to forward alerts

dockerymick committed:
Adding section on how to forward alerts

dockerymick committed:
Updates

clyang82 commented:
/assign @morvencao

morvencao commented on observability/customize_observability.adoc:
```diff
@@ -262,6 +263,60 @@ This means that there are six receivers in the environment.
+[#forward-alerts]
+== Forwarding alerts
+
+You can configure the `alertmanager-config` YAML file to forward alerts from managed clusters to the Alertmanager in the hub cluster. For example, configure the `alertmanager-config` YAML to receive alerts in Alertmanager on the hub cluster. Complete the following steps to set up forwarding:
```
@dockerymick 
For alerts forward from managed clusters to the hub cluster, end-users don't need to do anything. After the ACM 2.3 is installed with observability enabled, the alerts from the managed cluster with OCP version >=4.8 will be automatically forwarded to the hub cluster.

I think you are trying to add document for how to configure alert with external notification system, and how to do alert query/silence with amtool, that makes sense. But I have to mention that this is a generic topic for alertmanager, even we don't have alert forward feature, these functions are still available for the hub cluster's Alertmanager.

So I'm not sure if this is proper to be added under **forward-alerts** section.

dockerymick replied:
Thanks for your feedback @morvencao That's helpful that you clarified that setting up the external system is a generic topic for Alertmanager. With that info, I will remove the steps about the configuration to send alerts to an external system. 

I recommend that we add a reference link the Prometheus Alertmanager [doc](https://prometheus.io/docs/alerting/latest/alertmanager/)

morvencao replied:
That's what I was thinking, thanks, the change looks good to me.

morvencao replied:
nit: others looks good except this scentence:

> You can configure the `alertmanager-config` YAML file to forward alerts from managed clusters to the Alertmanager in the hub cluster.

the `alertmanager-config` YAML file is mainly used to configure alert with external notification system, not used to alert forward. If this could be change a bit, it will be better.

dockerymick replied:
@morvencao Ok got it. Since we aren't providing instructions on configurations for external notification systems, I recommend that we remove this line and replace it with the following statement: 

After you enable observability, alerts from your OCP managed cluster are automatically sent to the hub cluster. You can use the `alertmanager-config` YAML file to configure alerts with an external notification system. Complete the following steps to access the `alertmanager-config` YAML file:

morvencao replied:
That's will be perfect! @dockerymick

dockerymick committed:
Update customize_observability.adoc

Remove instruction on alertmanager configuration to set up forwarding for external systems. Added reference link

dockerymick committed:
Update observability_enable.adoc

morvencao reviewed (commented):
/lgtm

dockerymick commented on observability/customize_observability.adoc:
```diff
@@ -262,6 +263,30 @@ This means that there are six receivers in the environment.
++
+----
+need 
+  example
```
@morvencao may you provide a snippet of the alertmanager-config file?

morvencao replied:
I found a sample configuration YAML snippet, hopefully this helps:
```
global:
  slack_api_url: '<slack_webhook_url>'

route:
  receiver: 'slack-notifications'
  group_by: [alertname, datacenter, app]

receivers:
- name: 'slack-notifications'
  slack_configs:
  - channel: '#alerts'
    text: 'https://internal.myorg.net/wiki/alerts/{{ .GroupLabels.app }}/{{ .GroupLabels.alertname }}'
```

dockerymick replied:
Thanks @morvencao this does help, is there more information that the user should know about? maybe the `kind` for the YAML and namespace info ?

morvencao replied:
@dockerymick The alertmanager configuration is different from the kubernetes resources, it doesn't contains `kind` and `apiVersion` fields.

dockerymick replied:
ok got it, thanks for confirming. Going to move forward and merge this

dockerymick committed:
Update customize_observability.adoc

Made some updates

dockerymick committed:
Update customize_observability.adoc

Update after second review

dockerymick committed:
Update customize_observability.adoc

add part of the YAML example

morvencao reviewed (commented):
/lgtm

dockerymick committed:
Update customize_observability.adoc

small update
